addons/account/models/account_journal_dashboard.py

```diff
@@ -91,7 +93,7 @@ def _query_has_sequence_holes(self):
     def _compute_has_sequence_holes(self):
         has_sequence_holes = set(journal_id for journal_id, _prefix in self._query_has_sequence_holes())
         for journal in self:
-            journal.has_sequence_holes = journal.id in has_sequence_holes
+            journal.has_sequence_holes = journal.id in has_sequence_holes if journal.type == 'sale' else False
```

Comment:
Actually, this should even be only in the view

Reply:
Ok will try to put the condition in the xml then :)